awswrangler/catalog/_create.py

```diff
@@ -644,14 +644,42 @@ def create_database(
         args["Description"] = description
 
     try:
-        r = client_glue.get_database(Name=name)
-        if not exist_ok:
-            raise exceptions.AlreadyExists(f"Database {name} already exists and <exist_ok> is set to False.")
-        if description and description != r["Database"].get("Description", ""):
-            client_glue.update_database(**_catalog_id(catalog_id=catalog_id, Name=name, DatabaseInput=args))
+        client_glue.get_database(Name=name, CatalogId=catalog_id)
```

Comment:
This one fails if the catalog_id is not specified since the default value is None... Says it must be a string and not None. Added a if else to call get_database with or without catalog_id.

Reply:
are you sure? https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.get_database

What you describe is not possible, having an optional parameter with no default value. Unless the default value is a string which would be super strange.

Reply:
What I meant is that if the default value of the create_database function from data_wrangler for catalog ID (None) is passed in as
`glue_client.get_database(Name="some_existing_database", CatalogId=None)` then it will give the error in this boto3 functions:
```
ParamValidationError: Parameter validation failed:
Invalid type for parameter CatalogId, value: None, type: <class 'NoneType'>, valid types: <class 'str'>
```
So I think we must make the call to get_database depending on the value of catalog_id, if it's None then we don't pass that argument like:
`glue_client.get_database(Name="some_existing_database")` and work. boto3 will use then the default catalog id.

Reply:
got it. Yeap I don't think there is a way around it.